features/step_definitions/cars_garage_steps.rb

```diff
@@ -27,6 +27,7 @@
 
 When("I submit filled in new car form") do
   fill_in 'car_custom_name', with: "Test car custom name"
+  sleep 2
```

Comment:
What purpose has this `sleep` been added for?

Reply:
to wait for load async job of js script for choosing year. Passing of this test previously was matter of luck, you can check that you have to always wait a little to load years. Sometimes the time is short, sometimes long enough to break test